### DISCUSSION TITLE
supports go script plug-ins like xray

### DISCUSSION THREAD

**Discussion by werwolfz** [projectdiscovery/nuclei#3677] [Ideas]
im a starter，sry；i hope nuclei can Supports passive scanning, like xray; supports go script plug-ins or other methods, such as some vulnerabilities that cannot be implemented using yaml, and can be customized.
or give me some advices to lear nuclei and help me to finish these functions，thanks a lot

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
